### PR TITLE
FIX Collapse standard operations. Tweak License statements

### PR DESCRIPTION
**Pull request by jmcanterafonseca**

### CODE REVIEW

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 1]
```diff
@@ -3,29 +3,7 @@ HOST: http://orion.lab.fiware.org/
...
- General Public License for more details.
-
- You should have received a copy of the GNU Affero General Public License
- along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
-
- For those usages not covered by this license please contact with
- iot_support at tid dot es
--->
-
-FIWARE-NGSI Simple API.
```
Comment (jmcanterafonseca) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: we no longer need this header, as we have the License statement

Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: To be coherent, we should apply the same change in the cookbook .apib, that also includes the FIWARE Open Specification statement.

File: doc/apiary/v2/fiware-ngsiv2-reference.apib [hunk 2]
```diff
@@ -33,7 +11,9 @@ Please send your comments to fiware-ngsi@lists.fiware.org
 In addition, note that a list of currently open discussions is available at
 https://github.com/telefonicaid/fiware-orion/issues/1022
 
-Please check the following [FI-WARE Open Specification Legal Notice (essential patents license)](http://forge.fiware.org/plugins/mediawiki/wiki/fiware/index.php/FI-WARE_Open_Specification_Legal_Notice_%28essential_patents_license%29) to understand the rights to use this specification.
+This specification is licensed under the
+[FI-WARE Open Specification License]
```
Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: FI-WARE -> FIWARE (please do the same change also in the cookbook .apib)

File: doc/apiary/v2/fiware-ngsiv2-reference.apib (hunk 1 above)
Comment (jmcanterafonseca) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: that would be part of another PR, ok?

Comment (fgalan) [inline doc/apiary/v2/fiware-ngsiv2-reference.apib]: NTC

**Comment by fgalan** [comment]
LGTM
